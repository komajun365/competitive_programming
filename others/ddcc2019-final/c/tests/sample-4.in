-0.2018 -0.1190 0.6858
-0.5000 -0.7777
3
