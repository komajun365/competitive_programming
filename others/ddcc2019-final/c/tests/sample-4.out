0.266193331431572
0.767297626518897
1
1
